recipes/pycallgraph2/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pycallgraph2" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycallgraph2-{{ version }}.tar.gz
+  sha256: f5d02c1ea446cd7051ad886fc61c6698c1d1f6b759ac5e32ddec5fab153b73c8
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - pytest
```

Comment:
This is probably not needed but if they added it in the setup.py and the package checks for it you probably cannot removed it. Do you want to try to remove it to see how it goes?

Reply:
Sure, I'll do that.

Reply:
@ocefpaf Looks like it didn't work? Should I revert it?